content/version/2/1/code_of_conduct.es.md

```diff
@@ -3,87 +3,85 @@ version = "2.1"
 
-* El uso de lenguaje o imágenes sexualizadas, y aproximaciones o
-  atenciones sexuales de cualquier tipo
-* Comentarios despectivos (_trolling_), insultantes o derogatorios, y ataques personales o políticos
-* El acoso en público o privado
-* Publicar información privada de otras personas, tales como direcciones físicas o de correo
-  electrónico, sin su permiso explícito
-* Otras conductas que puedan ser razonablemente consideradas como inapropiadas en un
-  entorno profesional
+- Uso de lenguaje o imágenes de contenido sexual, así como insinuaciones o acercamientos sexuales no deseados.
+- Comentarios despectivos (*trolling*), insultantes o denigrantes, y ataques personales o políticos.
+- Acoso, ya sea en espacios públicos o privados.
+- Divulgación de información personal de otras personas (como direcciones físicas o electrónicas) sin su consentimiento explícito.
+- Cualquier otra conducta que pueda considerarse inapropiada en un entorno profesional.
 
-## Aplicación de las responsabilidades
+## Responsabilidades de aplicación
 
-Los administradores de la comunidad son responsables de aclarar y hacer cumplir nuestros estándares de comportamiento aceptable y tomarán acciones apropiadas y correctivas de forma justa en respuesta a cualquier comportamiento que consideren inapropiado, amenazante, ofensivo o dañino.
+El equipo de administración de la comunidad es responsable de clarificar y hacer cumplir este Código de Conducta, y tomará las medidas correctivas que considere apropiadas y equitativas ante cualquier comportamiento que se considere inadecuado, amenazante, ofensivo o perjudicial.
 
-Los administradores de la comunidad tendrán el derecho y la responsabilidad de eliminar, editar o rechazar comentarios, _commits_, código, ediciones de páginas de wiki, _issues_ y otras contribuciones que no se alineen con este Código de Conducta, y comunicarán las razones para sus decisiones de moderación cuando sea apropiado.
+Este equipo tendrá el derecho y la responsabilidad de eliminar, editar o rechazar comentarios, *commits*, código, ediciones de *wiki*, *issues* y otras contribuciones que no se ajusten a este Código de Conducta. Cuando sea pertinente, se comunicarán los motivos de las decisiones de moderación.
 
 ## Alcance
 
-Este código de conducta aplica tanto a espacios del proyecto como a espacios públicos donde un individuo esté en representación del proyecto o comunidad. Ejemplos de esto incluyen el uso de la cuenta oficial de correo electrónico, publicaciones a través de las redes sociales oficiales, o presentaciones con personas designadas en eventos en línea o no.
+Este Código de Conducta se aplica tanto dentro de los espacios del proyecto como en cualquier otro espacio donde una persona esté representando oficialmente a la comunidad. Algunos ejemplos incluyen el uso de una cuenta de correo institucional, publicaciones en redes sociales oficiales o la participación como representante designado/a en eventos presenciales o virtuales.
```

Comment:
```suggestion
Este Código de Conducta se aplica tanto dentro de los espacios del proyecto como en cualquier otro espacio donde una persona esté representando oficialmente a la comunidad. Algunos ejemplos incluyen el uso de una cuenta de correo institucional, publicaciones en redes sociales oficiales o la participación como representante designada en eventos presenciales o virtuales.
```

Reply:
Hey! Thanks for the suggestion and for looking over the text. I really appreciate your input!

My intention in using the equivalent of `designado/a` was to ensure the Code of Conduct is inclusive right from its wording. By using a gender-inclusive form, we make sure that any person, regardless of their gender, feels explicitly included and represented in that role.

Since this document aims to be a guide for the entire community, using language that embraces everyone helps reinforce an environment of equality and respect. It's a small detail, but I think it contributes significantly to the spirit of the code. What do you think of that approach?